pygac/gac_run.py

```diff
@@ -63,20 +67,38 @@ def check_file_version(filename):
         return main
 
 
+def str2scanline(string):
+    """Convert string to scanline.
 
-if __name__ == "__main__":
-    import sys
-    try:
-        filename = sys.argv[1]
-        start_line = sys.argv[2]
-        end_line = sys.argv[3]
-    except IndexError:
-        print "Usage: gac_run <filename> <start scan line number> <end scan line number>"
-        sys.exit(1)
-
-    reader = check_file_version(filename)
-    try:
-        reader(filename, start_line, end_line)
-    except ValueError:
-        print "Value error"
+    Make sure, the scanline is not negative.
+
+    Args:
+        string (str): String to be converted
+
+    Returns:
+        int: Scanline
+    """
+    integer = int(string)
```

Comment:
Yes, I use that here :) Since I want to add the check for negative values, I provide a separate method. And I didn't put the check in validate_args() because argparse can even tell you which argument is wrong.